class/SyncHandlers/PostHandler.php

```diff
@@ -57,6 +57,8 @@ protected static function map_data(array $data, int $entity_id)
         "post_opens_in_new_tab" => $data["OpensInNewTab"],
         "post_quote_text" => $data["QuoteText"],
         "post_quote_url" => $data["QuoteUrl"],
+        "post_metadata_title" => $data["MetaData"]['Title'] ?? "",
+        "post_metadata_description" => $data["MetaData"]['Description'] ?? "",
       ],
     ];
```

Comment:
The post handler seems to correctly map the details it receives from the API with the above change but we need to change other areas of the WP plugin as well in order to make this data get inserted in WP and used by the template.
The post data array created here is a temporary post data object, used to do the insertion of data into the WP system.

Namely,

- Relevant fields to store this data need to be added to 

https://github.com/passle/passle-sync-wordpress/blob/master/class/Models/PasslePost.php

and update the code where the post is actually inserted:

https://github.com/passle/passle-sync-wordpress/blob/master/class/SyncHandlers/SyncHandlerBase.php#L126

Reply:
- Model PHP and MD info updated.
- It was not necessary to update SyncHandlerBase.php#L126. The data was already inserted in the DB.
$postarr["meta_input"] is looped and inserted into wp_postmeta table.